linkerd/app/outbound/src/sidecar.rs

```diff
@@ -74,15 +90,92 @@ impl Outbound<()> {
 
                     http::Routes::Endpoint(Remote(ServerAddr(*orig_dst)), Default::default())
                 };
-                let routes = match (*parent).profile.clone() {
-                    Some(profile) => {
-                        let mut rx = watch::Receiver::from(profile);
-                        let init = mk_routes(&*rx.borrow_and_update());
-                        http::spawn_routes(rx, init, move |profile: &profiles::Profile| {
-                            Some(mk_routes(profile))
+
+                let mk_policy_routes = move |policy: &policy::ClientPolicy| {
+                    // A `ClientPolicy` does not have a single known logical
+                    // address, as it may consist of multiple distinct backends.
+                    // Therefore, just use the original destination address for now.
+                    // TODO(eliza): eventually, remove the logical addr field on
+                    // the policy route types when we use metadata for those
+                    // labels instead.
```

Comment:
Is this comment accurate? I don't understand it.

Reply:
The `HttpRoutes` types need to be constructed with a `LogicalAddr`, which is used for metrics labels etc. Unlike ServiceProfiles, we don't have a single authority to use for that logical address; the policy target is instead identified by its structured metadata. Currently, we use the original destination address for the `LogicalAddr`, but (as we discussed previously), we would like to remove the need to have `Param<LogicalAddr>` impls for the route types and generate metric labels and stuff based on the `Metadata` instead.

Does that make sense? I can try to reword the comment to make this clearer.

Reply:
tried to reword it in 584fd7703821d05de0677056c5f9c8f6a711a2e3, LMKWYT!